### PR TITLE
Draft of dummy storage

### PR DESCRIPTION
**Pull request by markusdregi**
Resolves: #1165

### CODE REVIEW

File: ert3/storage/__init__.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
+import ert3
```
Comment (oyvindeide) [inline ert3/storage/__init__.py:1]: Why is everything put in `__init__.py` files? Is it necessary to import everything always?

File: ert3/storage/__init__.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    storage_location = _generate_storage_location(workspace)
+    _assert_storage_initialized(storage_location)
+
+    with open(storage_location) as f:
+        storage = yaml.safe_load(f)
+
+    return storage.keys()
+
+
+def add_input_data(workspace, experiment_name, input_data):
```
Comment (oyvindeide) [inline ert3/storage/__init__.py:60]: This looks a lot like class properties to me, any reason it is not a class (goes for all these `add/get` functions)?

File: ert3/storage/__init__.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+    storage[experiment_name] = {}
+
+    with open(storage_location, "w") as f:
+        yaml.dump(storage, f)
+
+
+def get_experiment_names(workspace):
+    storage_location = _generate_storage_location(workspace)
+    _assert_storage_initialized(storage_location)
```
Comment (oyvindeide) [inline ert3/storage/__init__.py:52]: This is repeated many times, should perhaps be validated on the calling side? Or a decorator?

File: ert3/storage/__init__.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    if experiment_name not in storage:
+        raise KeyError(f"Cannot get input data, no experiment named: {experiment_name}")
+
+    if "input" not in storage[experiment_name]:
+        raise KeyError(f"No input data for experiment: {experiment_name}")
+
+    return storage[experiment_name]["input"]
+
+
+def add_output_data(workspace, experiment_name, output_data):
```
Comment (oyvindeide) [inline ert3/storage/__init__.py:98]: None of these functions are called in production code.

Comment (markusdregi) [reply ert3/storage/__init__.py]: Because that will be resolved in #1166

**Review by autt — APPROVED** [review]

**Pull request merged**